.serena/memories/project_overview.md

```
@@ -0,0 +1,34 @@
+# huerd - Project Overview
+
+## Purpose
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)